### ISSUE TITLE
Screen record notifications showing as (null)

### ISSUE THREAD

**Issue by mwoolweaver** [Baw-Appie/Axon#19]
iPhone 8 Plus
iOS 13.3.1 
checkra1n 

![1FC054A8-DC7E-4A80-A3C2-67F986E85B00](https://user-images.githubusercontent.com/5729320/76284056-3399e300-626a-11ea-8dfe-a8136629cc12.png)

**Comment by he1zn0erg** [comment]
Yeah, same issue for other ‚apps‘. That‘s because it‘s not an actual app that you could pull the name from.

Would be great if instead of null, the dev would replace it with „Clear all this apps notifications“ or sth like that.

**Comment by mwoolweaver** [comment]
@Baw-Appie this is still an issue on version 1.2. 

It could be as @he1zn0erg said
